pwa/src/templates/templateParts/formHeader/FormHeaderTemplate.module.css

```diff
@@ -28,6 +28,10 @@
   margin-inline-end: var(--gateway-ui-size-xs);
 }
 
+.buttonContentWrap {
```

Comment:
Class names should represent _what_ they are, and not _how_ they're going to look.

E.g.: `.buttonContentWrap` describes _how_ they're going to look (their content will be wrapped) and not _what_ they are.

A easier example for this would be:

`.redButton` <= says _how_ they will look 👎🏼 
`.deleteButton` <= says _what_ they are 👍🏼 

In this case, just `.button` might be best. **However**, see my other comments, I don't think we need this className at all.